src/frontend/src/components/public/PPQConnect/messages.ts

```diff
@@ -0,0 +1,27 @@
+export default {
```

Comment:
If we have API's like this, it is good to have a description key explaining where this might be used.
description: "After users fill in the PPQ form we show them their next steps. this API renders the next steps"

This may not be rendered in any output, but anyone receiving this object can understand where they might use this.